### DISCUSSION TITLE
Fixed IP

### DISCUSSION THREAD

**Discussion by Wipnose** [ESPresense/ESPresense#823] [Ideas]
An extra feature during the ESPresence Base Station setup could be the optional assignment for fixed IP.

**Comment by amiganutcase** [comment]
This can be done by setting a lease reservation inside your router. I've done that for my Tuya smart bulbs to use local Tuya integration in Home Assistant.

**Reply by Wheemer** [reply]
My providers router doesn't allow me properly to do that. So I can also see lots of benefit in this.

**Reply by ShonP40** [reply]
Check if it allows you to disable the DHCP server.
If it does, you can install AdGuard Home onto your Home Assistant server and use its DHCP server to reserve IPs.

But yea, I do agree that there should be a way to just configure a static IP on the ESPresense node itself.

**Reply by Wheemer** [reply]
Thanks, I have since gotten a fiber convertor to an asus router. So I have full control now. I removed my providers modem/router altogether.
